src/Deprecated/Engine/Resources/Strings.resx

```diff
@@ -1181,6 +1181,9 @@
   <data name="DefaultTargets" UESanitized="false" Visibility="Public">
     <value>[default]</value>
   </data>
+  <data name="DeprecatedEventSerialization" UESanitized="false" Visibility="Public">
+    <value>Usage of unsecure BinaryFormatter during serialization of custom event type '{0}'. This will be deprecated soon. Please use Extended*EventArgs instead. More info: https://aka.ms/msbuild/metadata-self-ref</value>
```

Comment:
This seems like wrong link - should be `https://aka.ms/msbuild/eventargs` instead?